src/backend/src/utils/change-requests.utils.ts

```diff
@@ -136,11 +136,15 @@ export const updateBlocking = async (
  */
 export const validateChangeRequestAccepted = async (crId: number) => {
   const changeRequest = await prisma.change_Request.findUnique({ where: { crId } });
+  const currentDate = new Date();
 
   if (!changeRequest) throw new NotFoundException('Change Request', crId);
   if (changeRequest.dateDeleted) throw new HttpException(400, 'Cannot use a deleted change request!');
   if (changeRequest.accepted === null) throw new HttpException(400, 'Cannot implement an unreviewed change request');
   if (!changeRequest.accepted) throw new HttpException(400, 'Cannot implement a denied change request');
+  if (!changeRequest.dateReviewed) throw new HttpException(400, 'Cannot use an unreviewed change request');
```

Comment:
I believe this is redundant, given line 143 already checks this, but check with @anthonybernardi

Reply:
@RChandler234 yeah, good point. I also thought this line acts the same as line 143 but had to add it since changeRequest.dateReviewed is a type of `null` or `Date`.

 For another approach, I was thinking of creating a separate variable after line 143, that equals to changeRequest.dateReviewed and cast it as the type of `Date`. I was not sure which way you guys prefer me to complete this ticket, but I'll also wait for @anthonybernardi feedback on this.

Reply:
ohh gotcha, this is fine with me then